site/server/app/com/fortysevendeg/exercises/utils/OAuth2.scala

```diff
@@ -24,7 +30,11 @@ object OAuth2 {
   }
 }
 
-class OAuth2Controller(ws: WSClient, userService: UserServices) extends Controller {
+class OAuth2Controller(
+    implicit
+    transactor: Transactor[Task],
```

Comment:
In other places we needed the transactor implicit or otherwise the compiler would complain, I called it `T` because we were merely proving its existence. In this case I actually use the transactor instance so called it `transactor`, I'll rename it to `T` for consistency.